addons/im_livechat/views/res_users_views.xml

```diff
@@ -23,9 +29,14 @@
             <field name="inherit_id" ref="base.view_users_form"/>
             <field name="arch" type="xml">
                     <xpath expr="//group[@name='messaging']" position="after">
+                        <field name="has_access_livechat" readonly="1" invisible="1"/>
                         <group name="livechat" string="Livechat"
-                            invisible="share">
+                            invisible="share or not has_access_livechat">
                             <field name="livechat_username"/>
+                            <field name="livechat_lang_ids" string="Online Chat Language"
+                                readonly="0"
```

Comment:
Isn't `readonly="0"` the default, especially if `readonly=False` is set in python?